packages/editor/src/components/widgets/table/footer/TableFooter.tsx

```diff
@@ -1,21 +1,31 @@
 import './TableFooter.css';
 import { IvyIcons } from '@axonivy/editor-icons';
 import { useReadonly } from '../../../../context';
-import type { ReactNode } from 'react';
+import { useState, type ReactNode } from 'react';
 import Button from '../../button/Button';
 
 export const TableFooter = (props: { children: ReactNode }) => {
   return <tfoot className='table-footer'>{props.children}</tfoot>;
 };
 
 export const TableAddRow = (props: { colSpan: number; addRow: () => void }) => {
+  const [isButtonHovered, setIsButtonHovered] = useState(false);
+
   const readonly = useReadonly();
   return (
-    <tr>
-      <th colSpan={props.colSpan} className='add-row'>
-        <Button icon={IvyIcons.Plus} onClick={props.addRow} disabled={readonly} aria-label='Add row' />
-      </th>
-    </tr>
+    <div className='add-row-container'>
+      <Button
+        icon={IvyIcons.Plus}
+        onClick={props.addRow}
+        disabled={readonly}
+        aria-label='Add row'
+        onFocus={() => setIsButtonHovered(true)}
+        onBlur={() => setIsButtonHovered(false)}
+        onMouseEnter={() => setIsButtonHovered(true)}
+        onMouseLeave={() => setIsButtonHovered(false)}
```

Comment:
why do something via js what css can do too?

Reply:
e.g.

```css
.add-row-container .button:hover ~ .add-line {
    background-color: var(--A300);
}
```

Reply:
Oh my God, I've been looking for this specific functionality in CSS, but couldn't find it😂